share/ipld/namespace_hasher.go

```diff
@@ -41,8 +40,7 @@ func (n *namespaceHasher) Write(data []byte) (int, error) {
 		return 0, fmt.Errorf("ipld: only one write to hasher is allowed")
 	}
 
-	ln, nln, hln := len(data), int(n.NamespaceLen), n.Hash.Size()
-	innerNodeSize, leafNodeSize := (nln*2+hln)*2, nln+appconsts.ShareSize
+	ln := len(data)
 	switch ln {
```

Comment:
```suggestion
	switch len(data) {
```

Reply:
we return ln in error so no

Reply:
Then we might also want to use it on line 56

Reply:
done